lib/otel/span-processor.js

```diff
@@ -242,6 +250,16 @@ module.exports = class NrSpanProcessor {
       },
       [ATTR_DB_STATEMENT]: () => {}
     }
+    // DynamoDB entity linking
+    if (span.attributes[ATTR_DB_SYSTEM] === 'dynamodb') {
+      // arn:aws:dynamodb:${attrs['aws.region']}:${accountId}:table/${attrs.collection}
+      //   from test/versioned/aws-sdk-v3/client-dynamodb.test.js:233
+      const region = span.attributes['aws.region'] ?? 'unknown'
+      const accountId = this.agent.config.cloud.aws.account_id ?? 'unknown'
```

Comment:
you also have to set this https://github.com/newrelic/node-newrelic/blob/main/lib/instrumentation/aws-sdk/v3/lambda.js#L34C37-L34C45